classes/MerchantAPI.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * Altapay module for Prestashop
+ *
+ * Copyright © 2020 Altapay. All rights reserved.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+require_once(_PS_MODULE_DIR_.'/altapay/lib/altapay/altapay-php-sdk/lib/AltapayMerchantAPI.class.php');
+
+/**
+ * Wrapper for interacting with ALTAPAY merchant API
+ *
+ * @author Ulrik Andersen ulrik.and@gmail.com
+ */
+class MerchantAPI
+{
+    /**
+     * @var AltapayMerchantAPI
+     */
+    private $api;
+    private $api_url;
+    private $api_username;
+    private $api_password;
```

Comment:
Please document property types:
```suggestion
    /** @var AltapayMerchantAPI */
    private $api;
    /** @var string */
    private $api_url;
    /** @var string */
    private $api_username;
    /** @var string */
    private $api_password;
```